docs/concepts/storage.md

```diff
@@ -3,3 +3,157 @@ sidebar_position: 3
 ---
 
 # Storage
+
+:::tip TLDR
+
+The punchline from this concepts doc is that Responsive Platform separates
+storage from compute as a means of improving operations without sacraficing
+transactional semantics at a neglible performance penalty.
+
+:::
+
+## History & Background
+
+Historically, Kafka Streams bundled storage locally on compute nodes by
+providing RocksDB based implementations of state stores. While storage on the
+compute nodes was not required to be durable (it could be recovered from
+a [changelog topic](https://developer.confluent.io/courses/kafka-streams/stateful-fault-tolerance/)
+in Kafka), performance necessitated storage to be persistent across restarts.
+The motivation for such a design was clear at the time: most deployments of
+Kafka Streams were running without access to the public cloud, and keeping
+the library free of external dependencies other than Kafka made it uniquely
+easy to deploy.
+
+The limitations of copuling compute with storage are well known: 
+1. Sizing your nodes becomes more of an art than a science, and many Kafka
+   Streams deployments in the wild tend to overprovision on both to ensure
+   that neither limitation is met.
+1. Scaling out becomes difficult since new nodes need to bootstrap storage
+   before they can become operational.
+1. Recovering from a fault or rebalancing is a challenge for the same reason
+   as above.
+
+:::info
+
+Curious to read more about tuning RocksDB state stores and contrasting it with
+Responsive Storage? Read our [blog post](https://responsive.dev/blog/separating-storage-and-compute)
+about it.
+
+:::
+
+Over time, the Kafka Streams developers have come up with elegant improvements
+to these problems: standby and warmup replicas, incremental rebalancing, sticky 
+task assignment... the list goes on.  Responsive takes a different approach to 
+solving this class of problems in Kafka Streams by improving the underlying 
+architecture to separate compute and storage - effectively eliminating these 
+concerns in one fell swoop.
+
+## Remote Storage
+
+
+At Responsive, we believe that reactive backends should power the most business-
```

Comment:
I feel we can drop the marketing angle from the doc. We should just say "We've built our storage solution to leverage ScyllaDB, which is a battle tested distributed database that provides: ..".

Reply:
I think we should also mention that with managed services in the cloud, the motivation for the original design is somewhat invalidated, creating an opening for our solution.